rskj-core/src/main/java/co/rsk/config/ConfigLoader.java

```diff
@@ -128,4 +156,66 @@ private Config getNetworkDefaultConfig(Config userConfig) {
         logger.info("Network not set, using mainnet by default");
         return ConfigFactory.load(MAINNET_RESOURCE_PATH);
     }
+
+    private static void verify(String keyPath, @Nullable ConfigValue expectedValue, ConfigValue actualValue, List<String> problems) {
+        Objects.requireNonNull(keyPath);
+        Objects.requireNonNull(actualValue);
+        ConfigValueType actualValueType = Objects.requireNonNull(actualValue.valueType());
+
+        if (expectedValue == null) {
```

Comment:
This is the original case? When a misspelled property will be warned... am i right?

Reply:
This seems to be my original case presented in #1192. Maybe a comment explaining how we got here can help.

Reply:
agree, I would add a comment here.

Reply:
Yes, the `verify` method is recursive one and `expectedValue` could be `null` during recursive iterations when an appropriate matching key path is to found.